packages/crew_profile_base.rb

```diff
@@ -11,6 +11,7 @@ class Crew_profile_base < Package
   source_url 'https://github.com/chromebrew/crew-profile-base.git'
   git_hashtag @_ver
 
+  depends_on 'git'
```

Comment:
Better to remain it in `core.rb`? (since it is required for git source functionality in Chromebrew)

Reply:
Without this dependency listed, `crew_profile_base` tries to install before `git` is installed. That breaks installs.